packages/governance/src/contractHelper.js

```diff
@@ -190,6 +191,9 @@ const facetHelpers = (zcf, paramManager) => {
     limitedCreatorFacet,
     governedApis = {},
   ) => {
+    // Far side-effects the object, and can only be applied once
+    const farGovernedApis = Far('governedAPIs', governedApis);
```

Comment:
_after thinking about this a bit, it's probably OK as is. But I'll leave some breadcrumbs..._

can we make it the caller's responsibility to mark it `Far`? Making the record in one place and marking it `Far` in another is odd.

One possibility is to make a new record...

```suggestion
    // Far side-effects the object, and can only be applied once
    const farGovernedApis = Far('governedAPIs', { ... governedApis });
```

Reply:
> One possibility is to make a new record...

Thanks. That's a much better answer.